mutiny-core/src/nostr/nwc.rs

```diff
@@ -162,6 +162,22 @@ impl NostrWalletConnect {
 
             // if we need approval, just save in the db for later
             if self.profile.require_approval {
+                // if the invoice has no amount, we cannot pay it
+                if invoice.amount_milli_satoshis().is_none() {
+                    log_warn!(
+                        node_manager.logger,
+                        "NWC Invoice amount not set, cannot pay: {invoice}"
+                    );
+                    return Ok(None);
+                }
+
+                // if we have already paid this invoice, skip it
```

Comment:
This should probably apply to both auto or manual nwc requests

Reply:
good call